lib/vrl/parser/src/lex.rs

```diff
@@ -1184,7 +1257,7 @@ mod test {
     #![allow(clippy::print_stdout)] // tests
 
     use super::{StringLiteral, *};
-    use crate::lex::Token::*;
+    use crate::lex::Token;
```

Comment:
Same here, re: unrelated change that I'd like to see reverted.

Reply:
I noticed you tagged me for a re-review. I assume these will be reverted before the merge?

Reply:
On it now!